File: routeMap/app/src/test/java/ru/android/zheka/gmapexample1/home/TestHomeModule.java
package ru.android.zheka.gmapexample1.home;

import dagger.Binds;
import dagger.Module;
import ru.android.zheka.coreUI.IActivity;
import ru.android.zheka.fragment.Home;
import ru.android.zheka.model.HomeModel;
import ru.android.zheka.model.IHomeModel;

@Module
public abstract class TestHomeModule {
    @Binds
//    @ActivityScope
    public abstract IHomeModel bindHomeModel(HomeModel model);

    @Binds
    public abstract IActivity bindActivity(Home context);

//    @Binds
//    public abstract IPanelHomeVM bindHomeVM(PanelHomeVM vm);
}

File: routeMap/app/src/test/java/ru/android/zheka/gmapexample1/home/TestHomeModule.kt
package ru.android.zheka.gmapexample1.home

import dagger.Binds
import dagger.Module
import ru.android.zheka.fragment.Home
import ru.android.zheka.fragment.IHome
import ru.android.zheka.model.HomeModel
import ru.android.zheka.model.IHomeModel

@Module
abstract class TestHomeModule {
    @Binds //    @ActivityScope
    abstract fun bindHomeModel(model: HomeModel?): IHomeModel?

    @Binds
    abstract fun bindActivity(context: Home?): IHome? //    @Binds
    //    public abstract IPanelHomeVM bindHomeVM(PanelHomeVM vm);
}